src/plugins/authentication/firebase/App.js

```diff
@@ -22,8 +22,8 @@ const App = {
     await firebase.auth().signInWithEmailAndPassword(email, password)
   },
   signup: async (email, password) => {
-    const user = await firebase.auth().createUserWithEmailAndPassword(email, password)
-    await user.user.sendEmailVerification()
+    const userCredential = await firebase.auth().createUserWithEmailAndPassword(email, password)
```

Comment:
good job on keeping the variable convention right. Thanks. 
"Leave the playground cleaner than you found it."
A quote that has had profound impact https://www.matheus.ro/2017/12/11/clean-code-boy-scout-rule/